vaas/vaas/router/forms.py

```diff
@@ -72,7 +71,7 @@ class Meta:
                 choices=tuple((action.action, action.name) for action in configuration.actions)
             ),
             'priority': PrioritySelect(
-                choices=tuple([(i, i) for i in range(1, 500)]),
+                choices=tuple([(i, i) if i != 0 else ('', '') for i in range(0, 500)]),
```

Comment:
What is the reason for that change?

Reply:
This was quick proposition of fixing priority calculation in admin forms. @smigacz will do it later.
I remove this solution in this PR.